src/main/java/com/example/crm/ResourceManager/ResourceManager.java

```diff
@@ -0,0 +1,79 @@
+package com.example.crm.ResourceManager;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+
+@Entity
+@Table(name = "crm_rm")
+public class ResourceManager {
+    @Id
+    @SequenceGenerator(name = "rm_sequence", sequenceName = "rm_sequence", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "rm_sequence")
+    private Long id;
+    private String name;
+    private String email;
+    private String password;
+
+    public ResourceManager() {
```

Comment:
There is no need to write default constructor, getter setters for each field. Use lombok annotations. 

Add something like @Getter @Setter @DefaultConstructor below @Table annotation then we can remove this entire code.